spec/features/navigation_spec.rb

```diff
@@ -47,16 +47,42 @@
       end
     end
 
-    it "can redirect a user to 404 error if they try to access any path with /merchant" do
+    it "can redirect a regular user to 404 error if they try to access any path with /merchant" do
```

Comment:
So glad you caught this! Now our tests are much better - and actually confirm what was required